credoai/evaluators/fairness.py

```diff
@@ -59,9 +53,15 @@ def __init__(
 
     def _validate_arguments(self):
         check_existence(self.metrics, "metrics")
-        check_data_instance(self.data, TabularData)
-        check_existence(self.data.sensitive_features, "sensitive_features")
-        check_artifact_for_nulls(self.data, "Data")
+        check_existence(self.data.X, "X")
+        check_existence(self.data.y, "y")
+        if not check_pandas(self.data.sensitive_features):
```

Comment:
is this redundancy? Sensitive features must always be pandas objects.

Reply:
Yes it is. I can remove it. Is the fact that it makes validation steps directly readable off of the evaluator's validation function a good enough reason to keep it?

Reply:
I removed it. I don't think we want redundant validation.